src/components/pages/Navigation/index.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { connect } from 'react-redux';
+
+import __spacing from 'settings/__spacing';
+import Logo from 'components/ui/Logo';
+import { WHITE_SMOKE, BRAND_PRIMARY } from 'settings/__color';
+import { hideNavigationMenu, getDashboardPage } from 'store/actions/navigation';
+
+interface NavigationProps {
+  shouldNavMenuShow: boolean;
+  hideNavMenu: () => void;
+  showDashboardPage: (page) => string;
+  history: {
+    push: (url: string) => void;
+  };
+}
+
+export function Navigation({
```

Comment:
This is a `HamBurgerMenu` not `Navigation`.

Reply:
Alright, implementing the feedback asap

Reply:
All done